src/main/java/seedu/address/model/property/exceptions/BuyerNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package seedu.address.model.property.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified property.
+ */
+public class BuyerNotFoundException extends RuntimeException {}
```

Comment:
Should this extend `ListableNotFoundException`? Same for the `PropertyNotFoundException`.